tsconfig.json

```diff
@@ -1,8 +1,113 @@
+    "frontend/src/components/VMigrationNotice.vue",
+    "frontend/src/components/VVariations.vue",
+    "frontend/src/components/VBackToSearchResultsLink.vue",
+    "frontend/src/components/VSnackbar.vue",
+    "frontend/src/components/VSourcesTable.vue",
+    "frontend/src/components/TableSortIcon.vue",
+    "frontend/src/components/VCopyButton.vue",
+    "frontend/src/components/VSketchFabViewer.vue",
+    "frontend/src/components/VScrollButton.vue",
+
+    "frontend/src/components/VAudioThumbnail/**.vue",
+    "frontend/src/components/VAudioTrack/layouts/**.vue",
+    "frontend/src/components/VCheckbox/**.vue",
+    "frontend/src/components/VContentLink/**.vue",
+    "frontend/src/components/VFilters/**.vue",
+    "frontend/src/components/VHomeGallery/**.vue",
+    "frontend/src/components/VIcon/**.vue",
+    "frontend/src/components/VSvg/**.vue",
+    "frontend/src/components/VIconButton/**.vue",
+    "frontend/src/components/VInputField/**.vue",
+    "frontend/src/components/VItemGroup/**.vue",
+    "frontend/src/components/VLicense/**.vue",
+    "frontend/src/components/VLogoLoader/**.vue",
+    "frontend/src/components/VMediaInfo/**.vue",
+    "frontend/src/components/VMediaTag/**.vue",
+    "frontend/src/components/VModal/**.vue",
+    "frontend/src/components/VPopover/**.vue",
+    "frontend/src/components/VRadio/**.vue",
+    "frontend/src/components/VRecentSearches/**.vue",
+    "frontend/src/components/VSkeleton/**.vue",
+    "frontend/src/components/VTabs/**.vue",
+    "frontend/src/components/VContentReport/**.vue",
+    "frontend/src/components/VContentSwitcher/VSearchTypeButton.vue",
+    "frontend/src/components/VContentSwitcher/VSearchTypeItem.vue",
+    "frontend/src/components/VErrorSection/VErrorImage.vue",
+    "frontend/src/components/VHeader/VFilterButton.vue",
+    "frontend/src/components/VHeader/VPageLinks.vue",
+    "frontend/src/components/VHeader/VFilterIconOrCounter.vue",
+    "frontend/src/components/VHeader/VSearchBar/**.vue",
+    "frontend/src/components/VImageDetails/VRelatedImages.vue",
+    "frontend/src/components/VSearchResultsGrid/VImageCell.vue",
+    "frontend/src/components/VSearchResultsGrid/VImageGrid.vue",
+    "frontend/src/components/LoadingIcon.vue",
+    "frontend/src/components/VPill.vue",
+
+    "frontend/src/pages/about.vue",
+    "frontend/src/pages/feedback.vue",
+    "frontend/src/pages/search-help.vue",
+    /**
+     * Some files from this section cannot be converted to TypeScript because they are
+     * used in non-TS contexts (like Tailwind configuration). Therefore they're left as
+     * "vanilla JavaScript" with JSDoc based type annotations. This allows us to check
+     * them with TypeScript without introducing syntax changes that are not compatible
+     * with other tools we rely on.
+     */
+    "frontend/src/constants/**/*",
+    "frontend/src/server-middleware/healthcheck.js",
+    /** End of non TypeScript files section */
+    "frontend/**/*.json"
```

Comment:
I'm not sure why we include json files in our tsconfig :thinking:

Reply:
Not sure either, but I looks like you added it in this PR:

https://github.com/WordPress/openverse/pull/951

I tried removing it locally and playing around with modifying json files, I didn't notice anything 🤷